sage-ios/SAGE/SAGE/AnnouncementsViewController.swift

```diff
@@ -88,7 +89,21 @@ class AnnouncementsViewController: UITableViewController {
     
     func getAnnouncements() {
         AnnouncementsOperations.loadAnnouncements({ (announcements) -> Void in
-            self.announcements = announcements
+            if let role = LoginOperations.getUser()?.role {
```

Comment:
you should delete this since we now know how to access the endpoints and get the proper data. this implementation also doesnt work with paginated data